docs/sql-deployment-configurations.md

```diff
@@ -188,9 +189,9 @@ You can start the worker as daemon process.
 
 ### Deploy a cluster on multiple nodes
 
-You can deploy a Pulsar SQL cluster or Trino cluster on multiple nodes. The following example shows how to deploy a cluster on three-node cluster.
+You can deploy a Pulsar SQL cluster or Trino cluster on multiple nodes. The following steps shows how to deploy a cluster on three-node cluster.
 
-1. Copy the Pulsar binary distribution to three nodes.
+#### Step 1: Copy the Pulsar binary distribution to three nodes.
```

Comment:
```suggestion
Step 1: Copy the Pulsar binary distribution to three nodes
```

It's not necessary to make them as headings because #### is not shown on the right ToC